### PR TITLE
57 update score on user guess integrate with guessing endpoint

### PR DESCRIPTION
**Pull request by larsuzh**
Not ready to be merged yet!
Wanted to receive some feedback on the current implementation and thoughts on introducing a new endpoint /updatescore on the user service which adds the achieved score to the current user (which we then access from the location-riddle service)

### CODE REVIEW

File: findme-users/src/UserService.py [hunk 1]
```diff
@@ -8,16 +8,19 @@ def __init__(self, user_repository):
         self.user_repository = user_repository
 
     def post_user(self, data, user_id):
-        return self.user_repository.post_user_to_db({**data, "user_id": user_id})
+        return (self.user_repository.post_user_to_db({**data, "user_id": user_id})
+                .dict(exclude={"scores"}))
```
Comment (v4lentin1879) [inline findme-users/src/UserService.py:12]: Could it make sense to create a DTO for the user? Excluding the scores all the time might be a bit confusing at some point

Comment (v4lentin1879) [reply findme-users/src/UserService.py:12]: I'm currently setting up swagger_ui: https://github.com/uzh-ase-fs24/backend/issues/63. Will continue on that tomorrow. For it to work efficiently, we should use Pydantic as often as possible. Therefore, it makes sense to rework this (I can also do it). You can see below how swagger looks like with nice Pydantic models.
image

**Comment by larsuzh** [comment]
functionality is implemented, @v4lentin1879 do you think we can merge this branch and take care of the dto issues you mentioned on your branch?

**Comment by v4lentin1879** [comment]
@larsuzh fine for me

**Review by v4lentin1879 — APPROVED** [review]

**Pull request merged**